src/lib/eZ/FieldType/RichText/RichTextStorage.php

```diff
@@ -57,6 +57,10 @@ public function storeFieldData(VersionInfo $versionInfo, Field $field, array $co
 
         $links = $xpath->query($xpathExpression);
 
+        if ($versionInfo->versionNo !== 1) {
```

Comment:
What's so special about version 1?

Reply:
no links to delete so the query is unnecessary because the content is just being created